snippet-mode/vars

```diff
@@ -5,7 +5,8 @@
 # --
 # -*- mode: snippet -*-
 # name: $1
-# key: ${2:trigger-key}${3:
+# contributor: ${2:`(user-full-name)`}
```

Comment:
Please move the contributor field above `name:`, I prefer that aesthetic fields be distinct from functional ones.

Reply:
@hlissner I've fixed.